geopandas/base.py

```diff
@@ -1152,10 +1152,10 @@ def voronoi_polygons(self, tolerance=0.0, extend_to=None, only_edges=False):
         By default, you get back a GeoSeries of polygons:
 
         >>> s.voronoi_polygons()
-        0     POLYGON ((-2 5, 1 2, -2 -1, -2 5))
-        1        POLYGON ((4 5, 1 2, -2 5, 4 5))
-        2    POLYGON ((-2 -1, 1 2, 4 -1, -2 -1))
-        3       POLYGON ((4 -1, 1 2, 4 5, 4 -1))
+        0        POLYGON ((-2 5, 1 2, -2 -1, -2 5))
+        1           POLYGON ((4 5, 1 2, -2 5, 4 5))
+        2       POLYGON ((-2 -1, 1 2, 4 -1, -2 -1))
+        3    POLYGON ((4 -1, 4 -1, 1 2, 4 5, 4 -1))
```

Comment:
I will also report this to upstream as the node (4, -1) is now duplicated here.

Reply:
It is actually a rounding issue only. It is `POLYGON ((4 -1, 3.9999999999999996 -1, 1 2, 4 5, 4 -1))`